docs/background/disaster-recovery.md

```diff
@@ -0,0 +1,56 @@
+# Disaster recovery
```

Comment:
Please add YAML front matter with a `description` field (like you already did in the howto).

Reply:
Ouch (and done).